### DISCUSSION TITLE
Uninstall udm-boot not working

### DISCUSSION THREAD

**Discussion by Troubadix** [unifi-utilities/unifi-common#424] [General]
Hi,

I receive the following error regarding udm-boot on my UDM SE

`E: The package udm-boot needs to be reinstalled, but I can't find an archive for it.`

I tried to remove the udm-boot but it fails:
`
dpkg -r udm-boot
dpkg: error processing package udm-boot (--remove):
 package is in a very bad inconsistent state; you should
 reinstall it before attempting a removal
Errors were encountered while processing:
 udm-boot
root@UDM-SE:/var/tmp#
`

Tried reinstalling but doesnt work too...

`
 dpkg -i udm-boot_1.0.5_all.deb
Selecting previously unselected package udm-boot.
(Reading database ... 44900 files and directories currently installed.)
Preparing to unpack udm-boot_1.0.5_all.deb ...
/var/lib/dpkg/tmp.ci/preinst: 19: /var/lib/dpkg/tmp.ci/preinst: /sbin/ssh-proxy: not found
dpkg: error processing archive udm-boot_1.0.5_all.deb (--install):
 subprocess new pre-installation script returned error exit status 127
/var/lib/dpkg/tmp.ci/postrm: 24: /var/lib/dpkg/tmp.ci/postrm: /sbin/ssh-proxy: not found
dpkg: error while cleaning up:
 subprocess new post-removal script returned error exit status 127
Errors were encountered while processing:
 udm-boot_1.0.5_all.deb
`

Any idea how to remove udm-boot or get rid of this error?

Thanks!

**Comment by iMonZ** [comment]
Same issue...

**Reply by boostchicken** [reply]
Get 1.0.5 from unifi.boostchicken.io also try a force flag

**Reply by boostchicken** [reply]
Also closing as this is not a unifi utils issue but a debain issue, google force package removal apt
